src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -94,44 +97,62 @@ public void sync() {
         localRepository.saveCache();
     }
 
-    // TODO: Move to constructor
-    public void initDatabase(Path launcherDir, Path gameDir) {
+    public void setupDirs(Path launcherDir, Path gameDir) {
```

Comment:
There are so many methods on the package manager now that it gets harder and harder to know what they are supposed to do without good documentation. 

- Is the whole set of `initLocalStorage`, `setupDirs`, `validateSources`, `initDatabase`, and `syncDatabase` the default way to spin up the a `PackageManager` instance?
- Do they have to be called in a specific order? 
- Are there any preconditions before calling each method?

Reply:
I realized the same too. :sweat_smile: I'm adding more docs in the upcoming commits. But the Package Manager might need a little bit of refactoring too, and it should be better done in a separate PR, once this one looks stable enough.